docs/varnish/vcl/varnish4.vcl

```diff
@@ -143,14 +143,14 @@ sub ez_purge {
         }
     }
 
-    if (req.method == "PURGE") {
+    if (req.method == "PURGE" || req.method == "PURGEKEYS") {
```

Comment:
varnish4.vcl can be dropped while on it, Varnish 4 support was dropped already in 2.5, but the file is here as 0.9 is also meant to work with 1.13. For master you can drop it.